src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -91,10 +91,12 @@ protected override void CheckBase(DiagnosticBag diagnostics)
                 return;
             }
 
+            Location baseLocation = null;
+
             // you need to know all bases before you can ask this question... (asking this causes a cycle)
             if (this.IsGenericType && !localBase.IsErrorType() && this.DeclaringCompilation.IsAttributeType(localBase))
             {
-                var baseLocation = FindBaseRefSyntax(localBase);
+                baseLocation ??= FindBaseRefSyntax(localBase);
```

Comment:
>??= [](start = 29, length = 3)

nit: `=` seemed fined. Why change?

Reply:
> nit: = seemed fined. Why change?

To make it safe to add something above the `if` later

---
In reply to: [456676550](https://github.com/dotnet/roslyn/pull/46053#discussion_r456676550) [](ancestors = 456676550)